docfx_project/docfx.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
1. we should probably exclude `Embrace_Tests`. i would imagine there's some setting to do that
2. none of the methods have any description as to what they do. do we need to add comments to the source to get this or where is that supposed to come from? if adding codes is the answer, please add comments in this PR or open a separate PR for that. you can get the descriptions for most (all?) methods from the [android API docs](https://embrace-io.github.io/embrace-android-sdk3/io/embrace/android/embracesdk/Embrace.html)

Reply:
1. can do.
2. Yes we can add comments. I'll keep the same PR. Thanks for pointing this out!